src/materialized/src/lib.rs

```diff
@@ -355,24 +355,39 @@ async fn serve_stash<S: mz_stash::Append + 'static>(
             Box::new(FilesystemSecretsController::new(secrets_storage))
         }
         Some(SecretsControllerConfig::Kubernetes {
-            context,
-            user_defined_secret,
-            user_defined_secret_mount_path,
-            refresh_pod_name,
+            ref context,
```

Comment:
Check out the streamlined single match version Sean suggested: https://github.com/MaterializeInc/materialize/pull/12236#discussion_r876322200
This worked quite nicely in my branch.

Reply:
will do!